var/spack/repos/builtin/packages/occa/package.py

```diff
@@ -50,13 +51,16 @@ class Occa(Package):
             description='Activates support for OpenCL')
     variant('coi',   default=False,
             description='Activates support for COI')
-    depends_on('cuda', when='+cuda')
-    
+    # depends_on('cuda', when='+cuda')
```

Comment:
This is really a part of a bigger discussion about CUDA.

The current cuda package is not very useful, but maybe this check can be done there so if we are on a Mac it tells the user to download from nvidia.developer.com and install manually?